sycl/doc/extensions/experimental/sycl_ext_oneapi_group_load_store.asciidoc

```diff
@@ -472,7 +489,7 @@ q.submit([&](sycl::handler& cgh) {
             auto offset = g.get_group_id(0) * g.get_local_range(0) *
                                        items_per_thread;
 
-            auto props = sycl_exp::properties{sycl_exp::contiguous_memory};
+            auto props = sycl_exp::properties{sycl_exp::contiguous_memory, sycl_exp::alignment<16>};
```

Comment:
This example does not have code to ensure that the `input` and `output` pointers are 16-byte aligned.  Can you change the calls above from `malloc_device` to `aligned_alloc_device`?

Reply:
Thanks, fixed that.